perfkitbenchmarker/disk.py

```diff
@@ -21,6 +21,11 @@
 
 from perfkitbenchmarker import resource
 
+STANDARD = 'standard'
+SSD = 'ssd'
+IOPS = 'iops'  # Provisioned IOPS (ssd) in AWS
+LOCAL = 'local'
```

Comment:
Talked offline - going to skip the separate local names - renaming ssd to remote_ssd is sufficient